apps/nextjs-website/src/app/robots.ts

```diff
@@ -0,0 +1,11 @@
+import { MetadataRoute } from 'next';
+
+export default function robots(): MetadataRoute.Robots {
+  return {
+    rules: {
+      userAgent: '*',
+      allow: '/',
+      disallow: '',
```

Comment:
For now, let's block the crawlers

```suggestion
      userAgent: '*',
      disallow: '/',
```